monitoring/storage.go

```diff
@@ -36,43 +36,50 @@ type StorageConfig struct {
 	Filesystems []string
 	// MinFreeBytes define minimum free volume capacity
 	MinFreeBytes uint64
-	// WatermarkWarning is the disk occupancy percentage that will trigger a warning probe
-	WatermarkWarning uint
-	// WatermarkCritical is the disk occupancy percentage that will trigger a critical probe
-	WatermarkCritical uint
+	// LowWatermark is the disk occupancy percentage that will trigger a warning probe
+	LowWatermark uint
+	// HighWatermark is the disk occupancy percentage that will trigger a critical probe
+	HighWatermark uint
 }
 
+// CheckAndSetDefaults validates this configuration object.
+// Config values that were not specified will be set to their default values if
+// available.
 func (c *StorageConfig) CheckAndSetDefaults() error {
 	var errors []error
 	if c.Path == "" {
 		errors = append(errors, trace.BadParameter("volume path must be provided"))
 	}
 
-	if c.WatermarkWarning > 100 {
-		errors = append(errors, trace.BadParameter("warning watermark must be between 0 and 100"))
+	if c.LowWatermark > 100 {
+		errors = append(errors, trace.BadParameter("low watermark must be 0-100"))
 	}
 
-	if c.WatermarkCritical > 100 {
-		errors = append(errors, trace.BadParameter("critical watermark must be between 0 and 100"))
+	if c.HighWatermark > 100 {
+		errors = append(errors, trace.BadParameter("high watermark must be 0-100"))
 	}
 
-	if c.WatermarkWarning == 0 {
-		c.WatermarkWarning = DefaultWarningWatermark
+	if c.LowWatermark == 0 {
+		c.LowWatermark = DefaultLowWatermark
 	}
 
-	if c.WatermarkCritical == 0 {
-		c.WatermarkCritical = DefaultCriticalWatermark
+	if c.HighWatermark == 0 {
+		c.HighWatermark = DefaultHighWatermark
+	}
+
+	if c.LowWatermark > c.HighWatermark {
+		errors = append(errors, trace.BadParameter("low watermark (%v) cannot be higher than high watermark (%v)", c.LowWatermark, c.HighWatermark))
```

Comment:
This is going to be an issue unless you always set both values in a single go. Would it make sense to have defaults be relative to each other like default low watermark is always 10% smaller than the high watermark?

Reply:
https://github.com/gravitational/satellite/pull/224#discussion_r436213986
I think we wanted to be able to configure the two values separately.

Ok, so instead of returning an error if LowWatermark > HighWatermark, we'll just set LowWatermark = HighWatermark. This way LowWatermark <= HighWatermark.